fips enabled spdmlib fails with "cannot find value `cert_chain_data` in this scope"

1fd6bebc introduced following error with fips-enabled spdm-rs:
```
error[E0425]: cannot find value `cert_chain_data` in this scope
   --> deps/spdm-rs/spdmlib/src/crypto/fips/asym_verify_st/mod.rs:144:35
    |
144 |         SpdmDer::SpdmDerCertChain(cert_chain_data),
    |                                   ^^^^^^^^^^^^^^^ not found in this scope

```

--- a/spdmlib/src/crypto/fips/asym_verify_st/mod.rs
+++ b/spdmlib/src/crypto/fips/asym_verify_st/mod.rs
@@ -141,7 +141,7 @@ fn ecdsa_verify(
     asym_verify::verify(
         hash_algo,
         asym_algo,
-        SpdmDer::SpdmDerCertChain(cert_chain_data),
+        SpdmDer::SpdmDerCertChain(certificate.as_slice()),
         msg,
         &sig,
     )

